src/main/java/ua/hillel/moneymanager/controller/UserController.java

```diff
@@ -0,0 +1,31 @@
+package ua.hillel.moneymanager.controller;
+
+import org.springframework.boot.autoconfigure.neo4j.Neo4jProperties;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.core.Authentication;
+import org.springframework.web.bind.annotation.*;
+import ua.hillel.moneymanager.dto.UserDto;
+import ua.hillel.moneymanager.repository.UserRepository;
+import ua.hillel.moneymanager.service.user.UserService;
+
+@RestController
+@RequestMapping
+public class UserController {
+    private UserService userService;
+
+    public UserController(UserService userService) {
+        this.userService = userService;
+    }
+
+    @PostMapping("/signup")
+    public ResponseEntity<UserDto> signup(@RequestBody UserDto userDto) {
+        return ResponseEntity
+                .ok(userService.registerNewUser(userDto));
+    }
+
+    @GetMapping("/info")
+    public ResponseEntity<UserDto> info(Authentication authentication) {
```

Comment:
Why do you need to explicitly pass `Authentication authentication`? Create a respective DTO, that contains fields required for getting this user from the database.